src/applications/simple-forms/21-4138/pages/nameAndDateOfBirth.js

```diff
@@ -9,7 +9,7 @@ import {
 import { getFullNameLabels } from '../helpers';
 
 /** @type {PageSchema} */
-export const nameAndDateOfBirthPage = {
+export const personalInformationPage = {
```

Comment:
Do we want the page to mismatch the file name like this?

Reply:
Good catch, fixed! 👍